.travis.yml

```diff
@@ -6,12 +6,17 @@ go:
   - 1.7
 
 install:
+  - export BUILD_READY=false
```

Comment:
@cclauss I would really want to address these as other issue(s); this PR has been open tool long IMO.  The Python 3 change should have been handled as separate PR as well.  We cannot keep prolonging a working/building PR with feature/scope creep.